spec/models/tag_spec.rb

```diff
@@ -11,4 +11,22 @@
       expect(tag).to respond_to(:jobs)
     end
   end
+
+  describe "instance methods" do
+    it ".average_salary returns average salary of all associated jobs" do
+      company = Company.create!(name: "ESPN")
+      job_1 = company.jobs.create!(title: "Developer", level_of_interest: 70, city: "Denver", salary: 30)
+      job_2 = company.jobs.create!(title: "Developer 2", level_of_interest: 70, city: "Denver", salary: 40)
+      job_3 = company.jobs.create!(title: "Manager", level_of_interest: 70, city: "Denver", salary: 50)
+
+      engineering = Tag.create!(name: "engineering")
+      tech = Tag.create!(name: "tech")
+
+      engineering.jobs << [job_1, job_2]
+      tech.jobs << [job_1, job_2, job_3]
+
+      expect(engineering.average_salary).to eq(35)
+      expect(tech.average_salary).to eq(40)
+    end
+  end
```

Comment:
Love the additional unit tests as well. One minor detail - `.class_method` vs `#instance_method` :)